docs/errors/jar-deprecated-jrubyversion.adoc

```diff
@@ -10,6 +10,9 @@ JRuby in an "embedded" scenario more reliable. Packing a `JRubyJar` being a
 fairly typical "embedded JRuby" use-case, there is certain functionality that
 relies on these improvements.
 
+
+NOTE: With JRuby/Gradle 2.0.0, JRuby versions 1.7.x are **completely** unsupported.
```

Comment:
I would rewrite this as: *As from JRuby-Gradle 2.0, the JRuby 1.7.x releases are no longer supported.*